opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/ProtocolResultHandler.java

```diff
@@ -235,6 +244,12 @@
         return mappedResult;
     }
 
+    private void setAspspRedirectTokenIfRequired(String authKey, FacadeRedirectResult mappedResult) {
```

Comment:
Immediate returning in condition will reduce cyclomatic complexity:
```
  if (!Strings.isNullOrEmpty(authKey)) {
         return;
   }
String toAspspRedirectToken = authService.generateToken(authKey, tppTokenProperties.getRedirectTokenValidityDuration());
mappedResult.setToken(toAspspRedirectToken);
```

Reply:
done